scripts/build/create-scratch-org.ps1

```diff
@@ -55,7 +55,7 @@ foreach($devHub in $devHubs) {
     }
 }
 
-$defaultUsername = (npx sfdx config:get defaultusername  --json | ConvertFrom-Json).result.value
+$defaultUsername = (npx sf config get target-dev-hub  --json | ConvertFrom-Json).result.value
```

Comment:
Same here, this should use `target-org` instead of `target-dev-hub`

```suggestion
$defaultUsername = (npx sf config get target-org  --json | ConvertFrom-Json).result.value
```